portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -132,4 +143,36 @@ private boolean getToChange(HttpServletRequest request) {
     String data = request.getHeader("change");
     return data != null && data.equals("true");
   }
+
+  /**
+   * @return the display name of the currently logged-in user. 
+   */
+  private String getDisplayName(String id) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("User").setFilter(new Query.FilterPredicate("id",
```

Comment:
[Nit] Always try to break at the highest semantic level across lines.  In other words if you have to split
`f(g(x, y)
`
prefer
```
f(
  g(x,y))
```

over 
```
f(g(x,
  y))
```